ga4gh/client.py

```diff
@@ -15,13 +15,26 @@
 requests.packages.urllib3.disable_warnings()
 
 
-class HTTPClient(object):
-
-    workaroundGoogle = 'google'
-
+class HttpClient(object):
```

Comment:
I tend to agree, but have tried to resist the urge to add my own preferences where they contradict PEP8. Want to make a PR against the STYLE.md with the argument, and update any existing code at the same time? Let's keep it as HTTPClient here for now to avoid confusing issues.